spec/fabricators/encryptables/credentials_fabricator.rb

```diff
@@ -5,12 +5,43 @@
 #  See the COPYING file at the top-level directory or at
 #  https://github.com/puzzle/cryptopus.
 
-Fabricator(:credential, from: 'Encryptable::Credentials') do
+Fabricator(:credential_all_attrs, from: 'Encryptable::Credentials') do
   transient :team_password
   name { Faker::Team.creature }
   cleartext_username { Faker::Internet.user_name }
   cleartext_password { Faker::Internet.password }
-  before_create do |account, attrs|
-    account.encrypt(attrs[:team_password])
+  cleartext_token { Faker::Internet.uuid }
+  cleartext_pin { Faker::Internet.ip_v4_address }
+  cleartext_email { Faker::Internet.email }
+  cleartext_custom_attr_label { Faker::Internet.user_name }
+  cleartext_custom_attr { Faker::Internet.password }
+  before_create do |encryptable, attrs|
+    encryptable.encrypt(attrs[:team_password])
+  end
+end
+
```

Comment:
create just one fabricator for credentials ....
add param to define which attrs should be used.
if no param, just create username, password

Reply:
Robin said that it is too much work, so I made a [follow-up ticket](https://github.com/puzzle/cryptopus/issues/742)